VotingApplication/VotingApplication.Web/Api/Models/DBViewModels/PollCreationRequestModel2.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using VotingApplication.Data.Model;
+
+namespace VotingApplication.Web.Api.Models.DBViewModels
+{
+    public class PollCreationRequestModel2
```

Comment:
Do we really need a new class for this? Especially one that's just extensions to an existing class? As we've commented out code that's not required in `PollController`, why can't we just do the same in `PollCreationRequestModel`? Classes with a "2" at the end are a bit worrying.